packages/code-du-travail-api/src/server/data_sources/code_du_travail_numerique.js

```diff
@@ -204,9 +204,76 @@ async function getDocsCount() {
     });
   // (res.hits.total && res.hits.hits[0]) || null);
 }
+/**
+ * Return related documents that match themes
+ * @param themes {Array} an array of theme to match
+ * @param id {string} @optional must_not_match
+ * @returns {Object} An elasticsearch response.
+ */
+async function searchRelatedDocument(themes, id) {
+  const match = themes.reduce((state, theme) => {
```

Comment:
This is looking like a `map` no ?

Reply:
looks like but not really, it return each item return an array
[A,B] => [[A',A''], [B',B'']]  
what I want is [A,B] => [A', A'', B', B'']